### ISSUE TITLE
hash_type wrong warning (textual)

### ISSUE THREAD

**Issue by lvg01** [saltstack/salt#33233]
### Description of Issue/Question
master/minion config with unset hash_type gives warning about hash_type not set as SHA256, e.g. for the minion: "IMPORTANT: Do not use md5 hashing algorithm! Please set "hash_type" to SHA256 in Salt Minion config!". The setting "hash_type: SHA256" fails starting the minion, correct setting is "hash_type: sha256" (in lowercase). Warning should be adjusted to the lowercase sha256

### Setup
hash_type: sha256

### Steps to Reproduce Issue
on both master/minions:
hash_type: SHA256 >>> minion fails te start
hash_type: sha256 >>> minion starts normally

### Versions Report
Salt Version:
           Salt: 2015.8.8.2
 
Dependency Versions:
         Jinja2: 2.7.2
       M2Crypto: 0.21.1
           Mako: Not Installed
         PyYAML: 3.10
          PyZMQ: 14.7.0
         Python: 2.7.5 (default, Oct 11 2015, 17:47:16)
           RAET: Not Installed
        Tornado: 4.2.1
            ZMQ: 4.0.5
           cffi: 0.8.6
       cherrypy: Not Installed
       dateutil: 1.5
          gitdb: Not Installed
      gitpython: Not Installed
          ioflo: Not Installed
        libgit2: Not Installed
        libnacl: Not Installed
   msgpack-pure: Not Installed
 msgpack-python: 0.4.6
   mysql-python: Not Installed
      pycparser: 2.14
       pycrypto: 2.6.1
         pygit2: Not Installed
   python-gnupg: Not Installed
          smmap: Not Installed
        timelib: Not Installed
 
System Versions:
           dist: redhat 7.2 Maipo
        machine: x86_64
        release: 3.10.0-327.13.1.el7.x86_64
         system: Red Hat Enterprise Linux Server 7.2 Maipo

**Comment by usbportnoy** [comment]
Happens on my minion also

```
Salt Version:
           Salt: 2015.8.8
 
Dependency Versions:
         Jinja2: 2.6
       M2Crypto: 0.21.1
           Mako: Not Installed
         PyYAML: 3.10
          PyZMQ: 14.0.1
         Python: 2.7.3 (default, Jun 22 2015, 19:33:41)
           RAET: Not Installed
        Tornado: 4.3
            ZMQ: 4.0.4
           cffi: Not Installed
       cherrypy: Not Installed
       dateutil: 1.5
          gitdb: Not Installed
      gitpython: Not Installed
          ioflo: Not Installed
        libgit2: Not Installed
        libnacl: Not Installed
   msgpack-pure: Not Installed
 msgpack-python: 0.3.0
   mysql-python: 1.2.3
      pycparser: Not Installed
       pycrypto: 2.6.1
         pygit2: Not Installed
   python-gnupg: Not Installed
          smmap: Not Installed
        timelib: Not Installed
 
System Versions:
           dist: Ubuntu 12.04 precise
        machine: x86_64
        release: 3.2.0-29-generic
         system: Ubuntu 12.04 precise
```

**Comment by Ch3LL** [comment]
@lvg01  I am also able to replicate this on 2015.8.8.2. Agreed this needs to change in the warning to make sure there is no confusion. Thanks! :)

```
➜  ~ grep hash /etc/salt/minion
hash_type: SHA256
```

```
[INFO    ] Minion is starting as user 'root'
[INFO    ] The Salt Minion is shut down
[ERROR   ] Minion failed to start
[WARNING ] ** Restarting minion **
[INFO    ] Sleeping random_reauth_delay of 9 seconds
```

**Comment by rallytime** [comment]
@lvg01 This is fixed at the HEAD of 2015.8 with PR #32360 and will be available in the upcoming 2015.8.9 release. :)
